bibliopixel/animation/sequence.py

```diff
@@ -1,14 +1,29 @@
 from . import collection
+import random as rand
 
 
 class Sequence(collection.Collection):
+    def __init__(self, layout, random=False, **kwds):
```

Comment:
I really have to push back here.

I like the idea of randomization a lot.  But you have a bad habit of adding code into an existing class, instead of creating a new class.

You've added either one or two independent features, I'm not sure which, and you've glommed them into this one nice, simple base class, basically making further maintenance on this class quite difficult.

`Random` should be a new class, derived from `Sequence` or even `Collection`.  There are all sorts of potential parameters on `Random` that make no sense for `Sequence` and vice versa.